src/nyc_trees/js/src/map.js

```diff
@@ -72,11 +72,17 @@ function fitBounds(map, bounds) {
 }
 
 function initBaseMap(map) {
-    var osmUrl = 'http://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png',
-        osmAttrib = 'Map data © <a href="http://openstreetmap.org">OpenStreetMap</a> contributors',
-        osm = new L.TileLayer(osmUrl, {maxZoom: zoom.MAX, attribution: osmAttrib});
+    var url = 'http://{s}.tile.stamen.com/tonerg/{z}/{x}/{y}.png',
+        options = {
+            subdomains: 'abcd',
```

Comment:
They get plugged in as values for ``{s}``. Browsers will often only load 2 resources at a time from a given domain. Using multiple subdomains allows for more tile requests in parallel.